package.json

```diff
@@ -11,41 +11,51 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build",
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
-    "start": "JOBS=1 ember serve",
-    "test": "ember test"
+    "start": "cross-env JOBS=1 ember serve -p 4200",
```

Comment:
I also added cross-env here for testing on windows machines (I occasionally give windows dev a try when I don't have my linux VM booted)